config/secrets.yml.sample

```diff
@@ -0,0 +1,43 @@
+default: &default
+  database:
+    name: datarepo
+    username: vagrant
+    password: changeme
+  ezid:
+    default_shoulder: doi:10.5072/FK2
+    user: apitest
+    password: apitest
+  fedora: &fedora
+    url: http://127.0.0.1:8080/fedora/rest
+    user: fedoraAdmin
+    password: fedoraAdmin
+  orcid:
+    app_id: 0000-0000-0000-0000
+    app_secret: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+  redis:
+    host: localhost
+    port: 6379
+  cas_endpoint_url: https://cas-dev.middleware.vt.edu
+
+development:
+  <<: *default
+  fedora:
+    <<: *fedora
```

Comment:
Before, this fedora block declaration was overriding the entire fedora block in default. Adding this fixes it.